### ISSUE TITLE
Fix broken GMP build

### ISSUE THREAD

**Issue by JohannesKauffmann** [JohannesKauffmann/ffmpeg-windows-build-script#1]
Currently, it fails at the configure step:
```
checking how to run the C preprocessor... x86_64-w64-mingw32-gcc -E
checking build system compiler x86_64-w64-mingw32-gcc... yes
checking for build system preprocessor... x86_64-w64-mingw32-gcc -E
checking for build system executable suffix... configure: error: Cannot determine executable suffix
```

Maybe just a newer version is needed...

**Comment by JohannesKauffmann** [comment]
This seems to be a WSL specific issue. It fails on both WSL Debian 10.8 and WSL Ubuntu 20.04. On native Ubuntu 20.04, it builds just fine.

A workaround could be to detect WSL by checking if `/proc/version` contains `Microsoft` and disable GMP.

**Comment by JohannesKauffmann** [comment]
Fixed since #2cf3ebef34f681a50e0804f5548aa39c77660924.
